<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TodoCreateDialog</class>
 <widget class="QDialog" name="TodoCreateDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>创建待办事项</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="priorityInput">
       <item>
        <property name="text">
         <string>低</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>中</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>高</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="categoryInput">
       <property name="placeholderText">
        <string>分类</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="deadlineInput">
       <property name="displayFormat">
        <string>yyyy-MM-dd HH:mm</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="titleInput">
     <property name="placeholderText">
      <string>标题</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="descriptionInput">
     <property name="placeholderText">
      <string>描述</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="clearButton">
       <property name="text">
        <string>清空</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
